base/db/R/dbfiles.R

```diff
@@ -347,7 +347,7 @@ dbfile.input.check <- function(siteid, startdate = NULL, enddate = NULL, mimetyp
       print(inputs)
       dbfile <- dbfile.check(
         type = "Input",
-        container.id = inputs$id,
+        container.id = inputs$parent_id,
```

Comment:
So I made this change to address a bug I was having with the parent and db file ids passed to dbfile.input.check, since there are two ids sometimes the function would try to run the query with the dbfile id instead of the parent id and this would trigger an error. I think I also added a fix on the ic_process side of things once I realized hat was happening, so I am not sure if this is necessary anymore.

Reply:
I do remember an issue where it was using the dbfile id when it was supposed to be using the input id (or vice versa) but that should be a separate issue from whether you use the input id (pointer to the current input) or the input parent_id (pointer to the input the current input was derived from)